packages/components/Button/src/Button.settings.ts

```diff
@@ -57,7 +58,6 @@ export const settings: IComposeSettings<IButtonType> = [
       focused: {
         tokens: {
           borderColor: 'buttonBorderFocused',
-          backgroundColor: 'buttonBackgroundHovered',
```

Comment:
Why does this even change any behavior?  This is set to the same value as the hovered state.

Reply:
I looked at the Figma file for WinUI Button redlines, and focusing is not supposed to have the hovered background color? Removing this setting only allows for the hover background during focus, when I hover over a already focused button.

Reply:
oh got it, thanks for explaining :)